functions-marc-curtis.js

```diff
@@ -0,0 +1,82 @@
+// Write a function named marco that returns "polo".
+// const functionName = () => {
+//     return 'polo'
+// }
+// console.log(functionName());
+
+// // Write a function named greeting that takes a name as an argument and returns "Welcome, <person's name here>!"
+// const greeting = (name) => {
+//     return `Welcome ${name}! `
+// }
+// console.log(greeting("Curits"));
+// // Write a function named oddOrEven that takes a number as an argument and returns whether the number is odd or even.
+// const oddOrEven = (number) => {
+//     if (number % 2 === 0){
+//         return "even"
+//     } else {
+//         return 'odd'
+//     }
+// }
+// console.log(oddOrEven(4));
+
```

Comment:
I have reviewed the other challenges. Your approach is great. Thank you for being mindful of the indentation.